packages/smooth_app/lib/pages/navigator/app_navigator.dart

```diff
@@ -285,18 +277,6 @@ class _SmoothGoRouter {
   static _SmoothGoRouter? _singleton;
   late GoRouter router;
 
-  // Indicates whether [_initAppLanguage] was already called
-  bool isInitialized = false;
-
-  void _initAppLanguage(BuildContext context) {
-    final UserPreferences userPreferences = context.read<UserPreferences>();
-    ProductQuery.setLanguage(context, userPreferences);
-    context.read<ProductPreferences>().refresh();
```

Comment:
I can't see the relationship with the current PR. We need a language, right, regardless of data migration?

Reply:
Oops I didn't see your comment.
The reason is just above:

`// The migration requires the language to be set in the app!`

Reply:
> The migration requires the language to be set in the app!

So does the app, so I hope that setLanguage in migration was redundant with a setLanguage in the app and that indeed we can get rid of setLanguage/migration.

Reply:
You're right, I've checked 1 year and a half ago and the code was placed elsewhere.
I will move the code.